pkg/plugins/scms/github/release_test.go

```diff
@@ -100,7 +289,7 @@ func TestSearchReleases(t *testing.T) {
 					mockedErr:   tt.mockedError,
 				},
 			}
-			releaseType := ReleaseType{}
+			releaseType := tt.releaseType
 			releaseType.Init()
```

Comment:
```suggestion
			tt.releaseType.Init()
```
No need to copy into another variable